runtime/core/utils/string.cc

```diff
@@ -153,7 +153,7 @@ std::string ProcessBlank(const std::string& str, bool lowercase) {
       std::wstring_convert<std::codecvt_utf8<wchar_t>, wchar_t> converter;
       std::wstring wsresult = converter.from_bytes(result);
       for (auto& c : wsresult) {
-        c = lowercase ? tolower(c, loc) : toupper(c, loc);
+        c = lowercase ? tolower(c, loc) : c;
```

Comment:
this changes the --nolowercase behavior, we could also pass another flag if that's what we want to do